### PR TITLE
Fix ai binding logs property

### PR DESCRIPTION
**Pull request by G4brym**
This re-adds a removed property as backwards compatibility, removed in the last release

### CODE REVIEW

**Comment by DaniFoldi** [comment]
Hi, I saw the workers-sdk test broken by the removal of this property. Just curious, if you can share, is there any particular reason the logs were removed/hidden in the first place?

Something like AI is much more complicated than say KV - there's not much to log in the case of KV (put or delete, that's it), but with AI, many things can happen, and logs were useful for me (and probably other folks who found them).

**Review by RamIdeas — APPROVED** [review]

**Comment by RamIdeas** [comment]
FYI we've decided to hold off upgrading workerd in wrangler until this PR lands and is released https://github.com/cloudflare/workers-sdk/pull/5752#issuecomment-2102288398

**Review by irvinebroque — APPROVED** [review]

**Pull request merged**
